apps/meeting/package-lock.json

```diff
@@ -1183,6 +1183,7 @@
       "version": "7.23.5",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.23.5.tgz",
       "integrity": "sha512-CgH3s1a96LipHCmSUmYFPwY7MNx8C3avkq7i4Wl3cfa662ldtUe4VM1TPXX70pfmrlWTb6jLqTYrZyT2ZTJBgA==",
+      "dev": true,
```

Comment:
why this is added to some dependencies?

Reply:
That means this dependencies is installed as dev dependency. This is not related to our change, they're the children dependencies of some package that we use. So most likely this means a direct package that we use changed this dependency to dev dependency in their implementation.